defaults:
  - door_scenario
  - override bopt@bopt_agent: bopt_gmm_dbopt

bopt_agent:
  base_accuracy: 0.65
  mean_range:  0.0
  sigma_range: 0.05
  opt_dims:
    means: null
    cvars:
      unary: null
      nary:  null
